config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  put "/guests/:id", to: "guest_rooms#create"
+
+  get "/rooms", to: "rooms#index"
 end
```

Comment:
lines 7 and 10 look good, but line 8 is not a correct RESTful route. A `put` is for modifying a resource, not creating a new one. We want a post here, the url could be `"/guests/:id/rooms"` or could do a post to `"guest_rooms"`.